Terminal.Gui/Views/ComboBox.cs

```diff
@@ -738,11 +744,15 @@ private void SetSearchSet ()
             return;
         }
 
+        _listview.SuspendCollectionChangedEvent ();
+
         // force deep copy
```

Comment:
Can you drop a `// PERF: Improve this` comment here?

This is fine as a short-term workaround to allow work to continue, but really should be a release blocker in this state.

Reply:
Also, if we merge this PR, can you keep the issue open or re-open it and assign to me after the merge, so it's on my radar?

Reply:
Maybe @mention me in the PERF comment, too, just for visibility?

Reply:
I can without any problems, but notice that I'm just ignoring it in the `ComboBox` because there is no need for the event to always be invoked just when opening and closing the list. In the `ListView`, the normal behavior continues to be used, that is, the event is invoked when there is a change in the data source, except when the user suspends it, of course.

Reply:
@dodexahedron done.

Reply:
Thanks.

Yeah, I'll take a deeper look at how it all fits together as part of a set of whole-library performance passes, since a lot of things are likely to involve more than just specific View types.